src/core/Form/Toggle/ToggleBase/ToggleBase.tsx

```diff
@@ -0,0 +1,37 @@
+import { ReactNode } from 'react';
+import { HtmlSpanProps } from '../../../../reset/HtmlSpan/HtmlSpan';
+
+export const baseClassName = 'fi-toggle';
+
+export interface ToggleBaseProps {
+  /** Controlled toggle-state - if provided, component will update only when this is explicitly changed */
+  checked?: boolean;
+  /** Default status of toggle when not using controlled 'checked' state
+   * @default false
+   */
+  defaultChecked?: boolean;
+  /** Custom classname to extend or customize */
+  className?: string;
+  /** Disable usage */
+  disabled?: boolean;
+  /**
+   * Label element content
+   */
+  children?: ReactNode;
+  /**
+   * aria-label for the HTML input-element,
+   */
+  'aria-label'?: string;
+  /**
+   * alternatively to aria-label you can define aria-labelledby with label-element id
+   */
+  'aria-labelledby'?: string;
+  /** Unique id
+   * @default uuidV4
+   */
+  id?: string;
```

Comment:
Maybe this should be updated to be like:

```JS
  /**
   * Unique id
   * If no id is specified, one will be generated automatically
   */
  id?: string;
```

Reply:
Changed this to match other components using AutoId